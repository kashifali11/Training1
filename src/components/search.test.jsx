import Store from "../redux/store.jsx";
import React from "react";
import { Provider } from "react-redux";
import renderer from "react-test-renderer";
import Search from "./search";

it("renders correctly", () => {
  const tree = renderer
    .create(
      <Provider store={Store}>
        <Search />
      </Provider>
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
