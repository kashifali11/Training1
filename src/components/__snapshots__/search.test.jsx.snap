// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="MuiContainer-root makeStyles-cont-1 MuiContainer-maxWidthLg"
>
  <div
    className="MuiFormControl-root MuiTextField-root MuiFormControl-fullWidth"
  >
    <div
      className="MuiInputBase-root MuiOutlinedInput-root MuiInputBase-fullWidth MuiInputBase-formControl"
      onClick={[Function]}
    >
      <input
        aria-invalid={false}
        autoFocus={false}
        className="MuiInputBase-input MuiOutlinedInput-input"
        disabled={false}
        onAnimationStart={[Function]}
        onBlur={[Function]}
        onChange={[Function]}
        onFocus={[Function]}
        placeholder="Search"
        required={false}
        type="text"
      />
      <fieldset
        aria-hidden={true}
        className="PrivateNotchedOutline-root-2 MuiOutlinedInput-notchedOutline"
        style={
          Object {
            "paddingLeft": 8,
          }
        }
      >
        <legend
          className="PrivateNotchedOutline-legend-3"
          style={
            Object {
              "width": 0.01,
            }
          }
        >
          <span
            dangerouslySetInnerHTML={
              Object {
                "__html": "&#8203;",
              }
            }
          />
        </legend>
      </fieldset>
    </div>
  </div>
  <button
    className="MuiButtonBase-root MuiButton-root MuiButton-contained MuiButton-containedPrimary"
    disabled={false}
    onBlur={[Function]}
    onClick={[Function]}
    onDragLeave={[Function]}
    onFocus={[Function]}
    onKeyDown={[Function]}
    onKeyUp={[Function]}
    onMouseDown={[Function]}
    onMouseLeave={[Function]}
    onMouseUp={[Function]}
    onTouchEnd={[Function]}
    onTouchMove={[Function]}
    onTouchStart={[Function]}
    style={
      Object {
        "marginTop": 16,
      }
    }
    tabIndex={0}
    type="button"
  >
    <span
      className="MuiButton-label"
    >
      search
    </span>
  </button>
</div>
`;
